src/shared/api/pluginsService/pluginsService.ts

```diff
@@ -0,0 +1,17 @@
+import { AragonBackendService } from '../aragonBackendService';
+import type { IDaoPlugin } from '../daoService';
+import type { IGetDaoPluginsByDaoParams } from './pluginsService.api';
+
+class PluginsService extends AragonBackendService {
+    private urls = {
+        pluginsByDao: '/v2/plugins/by-dao/:network/:address/details',
+    };
+
+    getDaoPluginsByDao = async (params: IGetDaoPluginsByDaoParams): Promise<IDaoPlugin[]> => {
```

Comment:
```suggestion
    getPluginsByDao = async (params: IGetDaoPluginsByDaoParams): Promise<IDaoPlugin[]> => {
```

maybe to simplify the name?

Reply:
renamed everywhere - removed "dao" before "pluginsByDao"